build.gradle

```diff
@@ -27,7 +27,7 @@ allprojects {
         appName = 'DestinationSol'
         gdxVersion = '1.9.8'
         roboVMVersion = '2.3.3'
-        nuiVersion = '3.0.0-SNAPSHOT'
+        nuiVersion = '3.1.0-SNAPSHOT'
```

Comment:
I guess this is still accurate? I don't see a v3.1 release yet. Thought maybe we had gone past that by now.